README.md

```diff
@@ -0,0 +1,30 @@
+# Installation
+This installation assumes you have a running instance of sensu-base and for ssl that you have your ssl-certs generated by the sensu-tool.
+```
+juju deploy sensu-client
+```
+The sensu-client requires 2 config options, password, which is the vhost password for the rabbitmq-server, exposed in juju status. The other one is rabbitmq, which is the privateip:port of the rabbitmq-instance. When using ssl, 2 extra config options must be set. Assuming you are in the sensu_ssl_tool directory:
```

Comment:
> the vhost password for the rabbitmq-server, exposed in juju status
> the privateip:port of the rabbitmq-instance

This is the sort of stuff that Juju should take care of using a relationship. Isn't there a relationship that does this? Can you create one?